site/_config.yml

```diff
@@ -32,7 +33,7 @@ archived_versions:
 - "4.1.0"
 - "4.0.0"
 latest_version: "4.9.0-SNAPSHOT"
-latest_release: "4.8.0"
+latest_release: "4.8.1"
```

Comment:
undo this change. so the version is not shown up at the menu.

Reply:
My idea was not to commit this PR but only the final one.
Anyway I am fine with this approach, changing this PR, I will rebase the second one

Reply:
done